.github/workflows/build.yml

```
@@ -13,7 +13,13 @@ jobs:
       - name: Setup
         uses: ./.github/actions/setup
       - name: Build
-        run: pnpm build
+        run: |
+          npx convex dev &        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)